lib/elixir/lib/regex.ex

```diff
@@ -582,8 +611,16 @@ defmodule Regex do
       iex> Regex.split(~r{a(?<second>b)c}, "abc", on: [:second], include_captures: true)
       ["a", "b", "c"]
 
+      iex> Regex.split(~r{-}, "-a-b--c", trim: true)
+      ["a", "b", "c"]
+
   """
-  @spec split(t, String.t(), [term]) :: [String.t()]
+  @spec split(t, String.t(), [
+          {:parts, pos_integer() | :infinity}
+          | {:trim, boolean()}
+          | {:on, capture_option()}
+          | {:include_captures, boolean()}
```

Comment:
Btw, you can use the keyword syntax. `[parts: ..., trim: boolean(), on: ...]`

Reply:
Oh nice! TIL, I didn't know about this syntax for specs.
Done in [09cf23c](https://github.com/elixir-lang/elixir/pull/13552/commits/09cf23c8c44e4241138cd9915577ac5590e9bd1a).